tests/test_fsclient.py

```diff
@@ -42,15 +38,108 @@
 DEFAULT_RC = {}
 rc = DEFAULT_RC
 
+
 # FileSystemClient methods tested here
 def test_is_alive():
-    expected = True  #filesystem is always alive!
+    expected = True  # filesystem is always alive!
     fsc = FileSystemClient(rc)
     actual = fsc.is_alive()
+
     assert actual == expected
 
-@pytest.mark.skip("Not written")
+
 def test_open():
+    fsc = FileSystemClient(rc)
+    fsc.open()
+
+    assert isinstance(fsc.dbs, type(defaultdict(lambda: defaultdict(dict))))
+    assert isinstance(fsc.chained_db, type(dict()))
+    assert not fsc.closed
+
+
+@pytest.mark.skip("Not written")
+def test_load_json():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_yaml():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_cif():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_database():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_json():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_yaml():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_cif():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_database():
+    pass
+
+
+def test_close():
+    fsc = FileSystemClient(rc)
+    fsc.close()
+
+    assert fsc.dbs is None
+    assert fsc.closed
+
+
+@pytest.mark.skip("Not written")
+def test_keys():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_collection_names():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_all_documents():
     pass
 
-# and so on.....
+
+@pytest.mark.skip("Not written")
+def test_insert_one():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_insert_many():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_delete_one():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_find_one():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_update_one():
+    pass
```

Comment:
please make sure all your files have a trailing eol at the end.  Set up your PyCharm (or whatever) to do it auomatically.